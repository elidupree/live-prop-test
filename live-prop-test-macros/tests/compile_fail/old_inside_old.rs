use live_prop_test::live_prop_test;

#[live_prop_test(postcondition = "old(old(true))")]
fn five(_: i32) -> i32 {
  5
}


fn main() {}
