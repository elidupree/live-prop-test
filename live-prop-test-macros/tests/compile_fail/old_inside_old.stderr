error: it doesn't make sense to use `old` inside another `old` expression
 --> $DIR/old_inside_old.rs:3:34
  |
3 | #[live_prop_test(postcondition = "old(old(true))")]
  |                                  ^^^^^^^^^^^^^^^^
